hexa/plugins/connector_accessmod/graphql/schema.graphql

```diff
@@ -322,6 +329,7 @@ extend type Mutation {
     deleteAccessmodFileset(input: DeleteAccessmodFilesetInput!): DeleteAccessmodFilesetResult!
     prepareAccessmodFileUpload(input: PrepareAccessmodFileUploadInput!): PrepareAccessmodFileUploadResult!
     prepareAccessmodFileDownload(input: PrepareAccessmodFileDownloadInput!): PrepareAccessmodFileDownloadResult!
+    prepareAccessmodFilesetVisualization(input: PrepareAccessmodFilesetVisualizationInput!): PrepareAccessmodFilesetVisualizationResult!
```

Comment:
Verbose, I know, but let's change that to `prepareAccessmodFilesetVisualizationDownload`